docs/overview/contributing/setting_up.rst

```diff
@@ -446,6 +456,23 @@ There are a number of such shell scripts in `ivy/run_tests_CLI`_:
 .. image:: https://github.com/unifyai/unifyai.github.io/blob/master/img/externally_linked/contributing/setting_up/setting_up_testing/pycharm_run_array_api_tests.png?raw=true
   :width: 420
 
+
+**For Windows users:**
+
+For Windows users, you may need to specify that the shell scripts should be run by :code:`sh`, which comes with Git. In the Terminal, prepend sh to the script commands like so:
+
+
+* To run :code:`run_tests.sh` on Windows, type :code:`sh ./run_tests_CLI/run_tests.sh` in the :code:`/ivy` directory.
+  This runs all tests in :code:`ivy/ivy_tests`.
+* To run :code:`test_array_api.sh` on Windows, type :code:`sh ./test_array_api.sh [backend] test_[submodule]`.
+  This runs all array-api tests for a certain submodule in a certain backend.
+* To run :code:`test_ivy_core.sh` on Windows, type :code:`sh ./run_tests_CLI/test_ivy_core.sh [backend] test_[submodule]` in the ivy directory.
+  This runs all ivy tests for a certain submodule in a certain backend in :code:`test_ivy/test_functional/test_core`.
+* :code:`test_ivy_nn.sh`, :code:`test_ivy_stateful.sh` are run in a similar manner to :code:`test_ivy_core.sh` on Windows.
+  Make sure to check the submodule names in the source code before running.
+
+The above instructions for running tests on Windows assume that you have installed Git and have access to the Git Bash terminal. If you do not have Git Bash, you can download it from the official Git website `<https://git-scm.com/downloads>_`.
```

Comment:
This link is still not rendering correctly, you may use an rst preview extension in your IDE to make sure it works, also check [sphinx docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links) here for more guidance, speaking of which, please keep using inline format.

> Use \`Link text <https://domain.invalid/>`_ for inline web links. If the link text should be the web address, you don’t need special markup at all, the parser finds links and mail addresses in ordinary text.